sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/Sample07_CrudOperations.cs

```diff
@@ -111,20 +113,21 @@ public async Task GetUpdateDeleteQueue()
         [Test]
         public async Task CreateTopicAndSubscription()
         {
-#if !SNIPPET
-            string topicName = Guid.NewGuid().ToString("D").Substring(0, 8);
-            string subscriptionName = Guid.NewGuid().ToString("D").Substring(0, 8);
-            string connectionString = TestEnvironment.ServiceBusConnectionString;
-            var client = new ServiceBusAdministrationClient(connectionString);
-#endif
+            string adminTopicName = Guid.NewGuid().ToString("D").Substring(0, 8);
+            string adminSubscriptionName = Guid.NewGuid().ToString("D").Substring(0, 8);
+            string adminConnectionString = TestEnvironment.ServiceBusConnectionString;
+
             try
             {
                 #region Snippet:CreateTopicAndSubscription
 #if SNIPPET
                 string connectionString = "<connection_string>";
                 string topicName = "<topic_name>";
-                var client = new ServiceBusManagementClient(connectionString);
```

Comment:
woops

Reply:
Looks like he moved it to line 130.

Reply:
Yeah; I found a few fun ones in both libraries while doing this.

Reply:
Yeah, I was referring to the prior use of the T1 client here @heaths